spec/features/course/assessment/submissions_viewing_spec.rb

```diff
@@ -72,23 +72,17 @@
     context 'As a Course Student' do
       let(:user) { create(:course_student, course: course).user }
 
-      scenario 'I can view my submitted and graded submissions' do
```

Comment:
No choice. I was stuck (for quite a while on these few specs) on figuring out how to write this in a more concise manner without losing any clarity but I realised sometimes you have to be more verbose >_<